packages/frontend/src/components/ElectionForm/ElectionSettings.tsx

```diff
@@ -27,25 +29,26 @@ export default function ElectionSettings() {
     const {t} = useSubstitutedTranslation(election.settings.term_type, {min_rankings, max_rankings});
 
     const [editedElectionSettings, setEditedElectionSettings] = useState(election.settings);
-    const [editedIsPublic, setEditedIsPublic] = useState(election.is_public);
     const [publicResults, setPublicResults] = useState(election.settings.public_results);
     const [ballotUpdates, setBallotUpdates] = useState(election.settings.ballot_updates);
-    const [ballotUpdatesDisabled, setBallotUpdatesDisabled] = useState(!ballotUpdatesConditionsMet);
-    const [publicResultsDisabled, setPublicResultsDisabled] = useState(false);
-    const [publicResultsDisabledMsg, setPublicResultsDisabledMsg] = useState(undefined);
-    const [ballotUpdatesDisabledMsg, setBallotUpdatesDisabledMsg] = useState(undefined);
+    const [ballotUpdatesDisabled, setBallotUpdatesDisabled] = useState(!ballotUpdatesConditionsMet || election.settings.public_results);
```

Comment:
nit, these 2 lines could use a comment explaining while public results is being passed in for ballot updates and vice versa. It would be easy for someone in the future to swap them assuming it's a typo